zad1/main.rb

```diff
@@ -1,13 +1,28 @@
 def sort_letters(string)
-
+    return string.split("").sort().join
```

Comment:
Jeśli nie podajemy argumentów do metody w Rubym to można spokojni ominąć nawiasy `()`